corehq/apps/cloudcare/static/cloudcare/js/formplayer/app.js

```diff
@@ -373,6 +373,12 @@ hqDefine("cloudcare/js/formplayer/app", function () {
                 }
             }
         );
+
+        window.addEventListener(
+            'beforeprint', function () {
+                let lastPanelElement = Utils.getLastClassEl('.panel.panel-default, .q.form-group');
+                if (lastPanelElement) {lastPanelElement.classList.add('last');}
```

Comment:
Style nitpick: It's typical in HQ code to put the block body on the next line. When I skim this code, seeing an `if` where the next line is a dedented closing bracket is surprising.